docs/docs/deploying-to-netlify.md

```diff
@@ -5,7 +5,7 @@ title: Deploying to Netlify
 This guide walks through how to deploy and host your next Gatsby site on [Netlify](https://www.netlify.com/).
 
 Netlify is an excellent option for deploying Gatsby sites. Netlify is a unified
-platform that automates your code to create performant, easily maintainable
+platform that automates your code to create performant, More easier maintainable
```

Comment:
Capitalization error. I think in this case you can omit the word without further replacement.